From 7eb714b8d69ad35646a8e810a8cf7fd95a968dc3 Mon Sep 17 00:00:00 2001
From: Sven Neumann <s.neumann@raumfeld.com>
Date: Tue, 6 Jul 2010 15:17:19 +0200
Subject: [PATCH] Flag string argument of GSSDPClient::message-received as static

Set the G_SIGNAL_TYPE_STATIC_SCOPE flag on the "from_ip" string
argument of the GSSDPClient::message-received signal.

Signed-off-by: Sven Neumann <s.neumann@raumfeld.com>
---
 libgssdp/gssdp-client.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/libgssdp/gssdp-client.c b/libgssdp/gssdp-client.c
index b5de2c3..713b1c8 100644
--- a/libgssdp/gssdp-client.c
+++ b/libgssdp/gssdp-client.c
@@ -428,7 +428,7 @@ gssdp_client_class_init (GSSDPClientClass *klass)
                               gssdp_marshal_VOID__STRING_UINT_INT_POINTER,
                               G_TYPE_NONE,
                               4,
-                              G_TYPE_STRING,
+                              G_TYPE_STRING | G_SIGNAL_TYPE_STATIC_SCOPE,
                               G_TYPE_UINT,
                               G_TYPE_INT,
                               G_TYPE_POINTER);
-- 
1.7.0.4

